Shared/UserAgent.swift

```diff
@@ -2,9 +2,30 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-import AVFoundation
+import WebKit
 import UIKit
 
+private extension WKWebView {
+    func evaluate(script: String, completion: @escaping (Any?, Error?) -> Void) {
```

Comment:
wrapping this in a Defered might make using this easier. I think with a Defered you can simply do `evalute("string").value` to force a Synchronous call. Unfortunate we cant do this any other way. User agent strings are pretty predictable. Could we build it ourselves manually?

Reply:
Deferred says `value` is not for production code, and it is removed from any version newer than what we have. We could also look at just putting in the UA manually yeah.